ros2node/ros2node/api/__init__.py

```diff
@@ -57,48 +65,51 @@ def get_node_names(*, node, include_hidden_nodes=False):
     ]
 
 
-def get_topics(remote_node_name, func):
+def get_topics(remote_node_name, func, include_hidden_topics=False):
     node = parse_node_name(remote_node_name)
     names_and_types = func(node.name, node.namespace)
     return [
         TopicInfo(
             name=t[0],
             types=t[1])
-        for t in names_and_types]
+        for t in names_and_types if include_hidden_topics or not _is_hidden_name(t[0])]
 
 
-def get_subscriber_info(*, node, remote_node_name):
-    return get_topics(remote_node_name, node.get_subscriber_names_and_types_by_node)
+def get_subscriber_info(*, node, remote_node_name, include_hidden=False):
+    return get_topics(
+        remote_node_name, node.get_subscriber_names_and_types_by_node, include_hidden)
```

Comment:
Keyword arguments should be passed with the explicit keyword. Same below.

Maybe the called function should force that with `, *, include_hidden_topics=False`.

Reply:
Done: 2367227